lib_test/mirage/unikernel.ml

```diff
@@ -29,36 +29,36 @@ module Server (C: Mirage_console_lwt.S) = struct
   let rec read_all c t =
     VX.read t
     >>= function
-    |`Eof -> C.log c "EOF"; OS.Time.sleep 5.
-    |`Error _ -> C.log c "ERR"; OS.Time.sleep 5.
-    |`Ok buf ->
+    | Ok `Eof -> OS.Time.sleep_ns (5.0 *. 1e9 |> Int64.of_float) >>= fun () -> C.log c "EOF"
```

Comment:
you may be interested in the [duration](https://github.com/hannesm/duration) library - `Duration.of_sec 5` does exactly that (and is much more readable)

Reply:
Ah, yes!  A wonderful recommendation.  Thanks!